### PR TITLE
Upgrade versioning #11594 keichwa

### PR DESCRIPTION
**Pull request by keichwa**
# Description

use terminology users already know, such as "Server- Major version upgrade", "Server - Minor version upgrade" and "Server - Patch level update"

# Target branches

Which product version does this PR apply to?

- [ ] Manager-4.0
- [x] Manager-4.1
- [x] Manager-4.2
- [x] Uyuni

# Links

Fixes https://github.com/SUSE/spacewalk/issues/11594

### CODE REVIEW

File: .changelog [hunk 1]
```diff
@@ -6,6 +6,8 @@
 
 # Last packaged: 2020-07-02
 
+- In the Upgrade Guide, use now use Major, Minor, and Patch Level
```
Comment (Loquacity) [inline .changelog:9]:
```suggestion
- In the Upgrade Guide, use Major, Minor, and Patch Level
```

File: modules/upgrade/pages/proxy-intro.adoc [hunk 2]
```diff
@@ -16,20 +16,20 @@ ifeval::[{suma-content} == true]
...
+Major Version Upgrade (X Upgrade)::
 Upgrading to the next major version.
 For example, upgrading from 3.2 to 4.0 or to 4.1.
 See xref:upgrade:proxy-x.adoc[].
 
-Y upgrades::
+Minor Version Upgrade (Y Upgrade)::
 Upgrading to the next minor version.
-This is often referred to as a service pack migration.
+This is often referred to as a service pack migration or SP migration.
```
Comment (Loquacity) [inline modules/upgrade/pages/proxy-intro.adoc:26]:
```suggestion
This is often referred to as a service pack (SP) migration.
```

File: modules/upgrade/pages/server-intro.adoc [hunk 3]
```diff
@@ -5,19 +5,19 @@
...
 Upgrading to the next major version.
-For example, upgrading from 3.2 to 4.0.
+For example, upgrading from 3.2 to 4.0 or to 4.1.
 See xref:upgrade:server-x.adoc[].
 
-Y upgrades::
+Minor Version Upgrade (Y Upgrade)::
 Upgrading to the next minor version.
-This is often referred to as a service pack migration.
+This is often referred to as a service pack migration or SP migration.
```
Comment (Loquacity) [inline modules/upgrade/pages/server-intro.adoc:15]:
```suggestion
This is often referred to as a service pack (SP) migration.
```

File: modules/upgrade/pages/upgrade-overview.adoc [hunk 4]
```diff
@@ -18,20 +18,21 @@ ifeval::[{suma-content} == true]
...
 For example, upgrading from 4.0 to 4.1.
 
-Z upgrades::
+Patch Level Upgrade (Z Upgrade)::
 Upgrading within the same minor version.
 This is often referred to as a maintenance update.
-For example, upgrading from 4.0.0 to 4.0.2.
+For example, upgrading from 4.0.0 to 4.0.2 or from 4.1.0 to 4.1.1.
 
+// Which term do we want to use here and later?  upgrading or updating?
```
Comment (Loquacity) [inline modules/upgrade/pages/upgrade-overview.adoc:35]: It's an upgrade guide, let's just pick upgrade and stick to it.

**Comment by Loquacity** [comment]
When you push a new commit, that failing build will clear out.

File: modules/upgrade/pages/upgrade-overview.adoc (hunk 4 above)
Comment (keichwa) [reply modules/upgrade/pages/upgrade-overview.adoc]: ok, it's in.

**Pull request merged**